newIDE/app/src/Utils/GDevelopServices/Build.js

```diff
@@ -185,13 +185,15 @@ export const getBuild = (
 
 export const getBuilds = (
   getAuthorizationHeader: () => Promise<string>,
-  userId: string
+  userId: string,
+  gameId?: string
```

Comment:
should I create 2 functions to show the difference between `getUserBuilds` and `getGameBuild` ?

Reply:
Maybe, could be clearer! Not a big fan of multiplying the number of optional parameters, it's almost always hard to understand. Better either :
- do another function (clearer)
- pass an object as argument for the options (but we don't need this flexibility)